src/magic_folder/cli.py

```diff
@@ -118,21 +118,21 @@
     read_config,
 )
 
-from ..frontends.magic_folder import (
+from magic_folder.frontends.magic_folder import (
```

Comment:
Can this be `from .frontends...`?  I'm not dead set on this.  I don't think there's a clear winner between absolute and explicit relative imports in the general case.